src/main/java/hudson/plugins/ws_cleanup/WsCleanup.java

```diff
@@ -251,7 +252,8 @@ public boolean needsToRunAfterFinalized() {
     public boolean isMatrixProject(Object o) {
         return o instanceof MatrixProject;
     }
-    
+
+    @Symbol("wsCleanup")
```

Comment:
No strong opinion. Or maybe `cleanWs()`? (to be shorter)
Let's see what others think.